capsules/src/process_console.rs

```diff
@@ -137,11 +139,19 @@ pub const READ_BUF_LEN: usize = 4;
 /// characters, limiting arguments to 25 bytes or so seems fine for now.
 pub const COMMAND_BUF_LEN: usize = 32;
 
+pub const DEFAULT_COMMAND_HISTORY_LEN: usize = 10;
+
 /// List of valid commands for printing help. Consolidated as these are
 /// displayed in a few different cases.
 const VALID_COMMANDS_STR: &[u8] =
     b"help status list stop start fault boot terminate process kernel panic\r\n";
 
+/// Escape character for ANSI escape sequences.
+const ESC: u8 = '\x1B' as u8;
+
+/// Null termiantor character
+const TERMINATOR: u8 = '\0' as u8;
```

Comment:
```suggestion
const EOL: u8 = '\0' as u8;
```

End of line sounds better :) .